frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -314,8 +314,8 @@ describeWithToken("scenarios > admin > people", () => {
 
     modal().within(() => {
       cy.findByText(fullName, { exact: false });
```

Comment:
@ranquild, I think you built this unsubscribe feature — I'm trying to make some small adjustments to the text in the modal, but this test is failing now that I've added the user's name to the title of the modal. Can you tell my why this check was here in the first place, and if it's okay to remove?

Reply:
Should be OK to change it to findAllByText